frontend_lib/src/component/EmojiReactions/EmojiReactionButtons.jsx

```diff
@@ -0,0 +1,252 @@
+    document.body.addEventListener('touch', this.handleBodyClick)
+  }
+
+  componentWillUnmount () {
+    document.body.removeEventListener('click', this.handleBodyClick)
+    document.body.addEventListener('touch', this.handleBodyClick)
+  }
+
+  render () {
+    const { props } = this
+
+    const emojiMartI18n = {
+      search: props.t('Search'),
+      clear: props.t('Clear'),
+      notfound: props.t('No Emoji Found'),
+      skintext: props.t('Choose your default skin tone'),
+      categories: {
+        search: props.t('Search Results'),
+        recent: props.t('Frequently Used'),
+        smileys: props.t('Smileys & Emotion'),
+        people: props.t('People & Body'),
+        nature: props.t('Animals & Nature'),
+        foods: props.t('Food & Drink'),
+        activity: props.t('Activity'),
+        places: props.t('Travel & Places'),
+        objects: props.t('Objects'),
+        symbols: props.t('Symbols'),
+        flags: props.t('Flags'),
+        custom: props.t('Custom')
+      },
+      categorieslabel: props.t('Emoji categories'),
+      skintones: {
+        1: props.t('Default Skin Tone'),
+        2: props.t('Light Skin Tone'),
+        3: props.t('Medium-Light Skin Tone'),
+        4: props.t('Medium Skin Tone'),
+        5: props.t('Medium-Dark Skin Tone'),
+        6: props.t('Dark Skin Tone')
+      }
+    }
+
+    return (
+      <>
+        <button
+          title={props.t('React')}
+          ref={this.buttonRef}
+          className={classnames('ReactionButton__buttonpicker', { active: this.state.pickerDisplayed })}
+          onClick={() => this.setState(
+            prev => ({
+              pickerDisplayed: !prev.pickerDisplayed
+            })
+          )}
+        >
+          <i className='far fa-smile' />
+        </button>
+        {(this.state.pickerDisplayed
+          ? (
+            <UncontrolledPopover
+              className="EmojiPickerPopover"
+              toggle={props.onCancel} // eslint-disable-line react/jsx-handler-names
+              placement='left'
+              isOpen
+              target={() => this.buttonRef.current}
+            >
+              <PopoverBody>
+                <Picker
+                  native
+                  emojiTooltip={true}
+                  title=''
+                  i18n={emojiMartI18n}
+                  onSelect={this.handleSelect}
+                />
+              </PopoverBody>
+            </UncontrolledPopover>
+          )
+          : undefined
+        )}
+      </>
+    )
+  }
+}
+
+const TranslatedEmojiPickerButton = translate()(EmojiPickerButton)
+
+function getReactionButtonTitle (props) {
+  const reactionList = props.reactionList
+
+  const you = (
+    reactionList[0].reaction_id === props.userReactionId
+      ? props.t('You')
+      : props.t('you')
+  )
+
+  const value = props.reactionList[0].value
+  const emoji = getEmojiDataFromNative(value, 'apple', emojiData)
```

Comment:
'apple'? The emoji style?

Reply:
Yes, getEmojiDataFromNative expects us to specify a set in which the emojis should be searched. I haven't tried to use the empty string, but maybe if it does not work I should had a comment explaining this indeed.

Reply:
the empty string actually works so I removed 'apple' there. Thanks!